docs/Reference/API-Methods.md

```diff
@@ -4484,6 +4484,43 @@ Returns full trace of all invoked opcodes of all transactions included in the bl
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### miner_changeTargetGasLimit
+
+Updates the target gas limit set using the [`--target-gas-limit`](CLI/CLI-Syntax.md#target-gas-limit)
```

Comment:
Sentence looks strange... I had to read it multiple times to finally understand...

```suggestion
Updates the target gas limit that was set using the [`--target-gas-limit`](CLI/CLI-Syntax.md#target-gas-limit)
```